src/BloomBrowserUI/bookEdit/js/editViewFrame.js

```diff
@@ -137,10 +110,154 @@ function initializeReaderSetupDialog() {
     document.getElementById('settings_frame').contentWindow.postMessage(fontMsg, '*');
 }
 
-
 /**
  * Called by C# after the setup data has been saved, following Save click.
  */
 function closeSetupDialog() {
     $('#synphonyConfig').dialog("close");
+}
+// "endregion" ReaderSetup dialog
```

Comment:
Feels out-of-place in editViewFrame.js. I'd expect the code for the add page button to be in its own (ts) file or possibly in one related to the page chooser window.